packages/itwinui-css/src/alert/alert.scss

```diff
@@ -54,11 +54,13 @@
 }
 
 .iui-alert-link {
-  @include anchor.iui-anchor;
-  @include anchor.iui-anchor-underline('on-initial');
   border-radius: var(--iui-border-radius-1);
   cursor: pointer;
   margin-inline-start: var(--iui-size-xs);
   user-select: none;
   white-space: nowrap;
 }
+
+.iui-anchor.iui-alert-link {
```

Comment:
Is `.iui-anchor` needed in combination with `.iui-alert-link`? Or is `.iui-alert-link` sufficient?

Reply:
It is needed in combination here because using just `.iui-alert-link` causes the style to be overwritten by the style in `.iui-anchor`, so a higher specificity is required.

Reply:
Done in latest [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/a6144abc9c3d1d69d0640a285709d1d72f47bd5b)

Reply:
Sorry, ignore the last comment here. Accidently put that comment on the wrong conversation.

Reply:
To confirm, is that commit link correct? Because that commit has no changes for `alert.scss`.

Reply:
Again, please ignore that comment. It was meant for this [conversation.](https://github.com/iTwin/iTwinUI/pull/1915#discussion_r1523450673)

Reply:
Oops, I missed reading https://github.com/iTwin/iTwinUI/pull/1915#discussion_r1526290640 😅

Reply:
All good. It was my mistake to begin with.

Reply:
The style ended up being completely removed in this [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/882a3b0dc56a27a8668d6b79dff78bdedeb4b9fd#diff-f4d2f744678b91bc8388e5cfed9d9326fa6f7061ae50ae71d5b542a8b09c3389) It was removed because of this [conversation.](https://github.com/iTwin/iTwinUI/pull/1915#discussion_r1526311053)